### DISCUSSION TITLE
Proxy-Cache status Bypass

### DISCUSSION THREAD

**Discussion by ssa7090** [Kong/kong#11081] [Help]
Hi, I should test Kong Gateway for my bachelor-thesis and I have installed the version 3.3. via the quickstart. All the configurations of the default gateway I did via Kong Manager and I did it the same way described in the guides. But for the proxy-caching I get always the status Bypass? I checked the content-type of the response headers which are text/plain;charset=UTF-8. In the configuration I have 3 rows inwhich I have the mentioned two and in the last application/json - so that all three types can be considered. But still I have the status bypass.

Can someone help me how I can solve this problem?

**Comment by raselSoftNode** [comment]
@ssa7090 did you find the solution? i have the same problem like you.

**Comment by flowdopip** [comment]
please share your configurations, you mave have the cache-control: true enabled and if the upstream is not answering with the cache headers, it will not work
